http.go

```diff
@@ -192,13 +195,16 @@ var bufferPool = sync.Pool{
 }
 
 func (h *httpGetter) Get(context Context, in *pb.GetRequest, out *pb.GetResponse) error {
-	u := fmt.Sprintf(
-		"%v%v/%v",
-		h.baseURL,
-		url.QueryEscape(in.GetGroup()),
-		url.QueryEscape(in.GetKey()),
-	)
-	req, err := http.NewRequest("GET", u, nil)
+	u, err := url.Parse(h.baseURL)
```

Comment:
Rather than parsing the URL on each Get, perhaps h.baseURL should be a *url.URL?